src/emulator/gxm/include/gxm/types.h

```diff
@@ -4,12 +4,13 @@
 #include <condition_variable>
 #include <glutil/object.h>
 #include <glutil/object_array.h>
+#include <rpcs3/BitField.h>
```

Comment:
We probably need a public dependency added to gxm/CMakeLists.txt.

Reply:
I've already done that haven't I?
`target_link_libraries(gxm PUBLIC crypto glutil mem sdl2 rpcs3 threads vita-headers)`